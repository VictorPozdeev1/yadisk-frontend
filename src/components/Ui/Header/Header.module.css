.logoImage {
  width: 30%;
  margin-bottom: 0;
  height: auto;
  margin-top: 1em;
}
