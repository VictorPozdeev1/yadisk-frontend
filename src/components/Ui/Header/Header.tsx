import React, { FC } from "react";
import style from "./Header.module.css";

const Header: FC<any> = () => {
  return <div className={style["wrapper"]} data-testid="header_test">Header</div>;
};

export default Header;
