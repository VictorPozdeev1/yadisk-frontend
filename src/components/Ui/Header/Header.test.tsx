import { render, screen } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom'
import Header from './Header'

describe('Header component', () => {
    test('Header render', () => {
        render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>
        )
    })

    test('Header snapshot', () => {
        const header = render(
            <BrowserRouter>
                <Header />
            </BrowserRouter>
        )

        expect(header).toMatchSnapshot();
    })
})