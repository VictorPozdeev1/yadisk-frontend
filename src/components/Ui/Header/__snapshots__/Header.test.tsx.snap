// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Header snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        data-testid="header_test"
        style="height: 100%; display: flex; flex-direction: column; align-items: center; text-align: left; position: relative;"
      >
        <div
          style="display: flex; justify-content: space-between; align-items: center; width: 100%; margin-top: 1em;"
        >
          <div
            style="display: flex; justify-content: space-between; align-items: center; width: 100%; margin-top: 1em;"
          >
            <img
              alt="Росатом"
              class="logoImage"
              src="/images/logo.png"
              style="margin: 0px auto;"
            />
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeMedium css-kw0y8f-MuiSvgIcon-root"
              data-testid="MenuIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      data-testid="header_test"
      style="height: 100%; display: flex; flex-direction: column; align-items: center; text-align: left; position: relative;"
    >
      <div
        style="display: flex; justify-content: space-between; align-items: center; width: 100%; margin-top: 1em;"
      >
        <div
          style="display: flex; justify-content: space-between; align-items: center; width: 100%; margin-top: 1em;"
        >
          <img
            alt="Росатом"
            class="logoImage"
            src="/images/logo.png"
            style="margin: 0px auto;"
          />
        </div>
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-colorSecondary MuiSvgIcon-fontSizeMedium css-kw0y8f-MuiSvgIcon-root"
            data-testid="MenuIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
